### PR TITLE
Creation of t_linkedin_create_account

### PR DESCRIPTION
**Pull request by braydenmarsh**
Hi Lauren and Eden, 

Here is my initial draft for the "How to Create a LinkedIn Account" task topic. 

Please let me know what you think.

I am mostly concerned with 
- is the shortdesc good? I wonder if I need more or less.
- Do you believe my steps are comprehensive enough or if they need more.
- Is my organization reasonable? Is there a better way to organize this information? I'm wondering about substeps mostly. 

Let me know what you think!

### CODE REVIEW

**Comment by lhmarlie** [comment]
Hi Brayden, yes, I think it looks pretty good. The shortdesc is short but it succinctly describes why a person might create an account. I think the steps look fine but it may not be until you create a ditamap and actually see the output that you want to go back and redo some things.

File: tasks/t_linkedin_create_account.dita [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <step>
+        <cmd>Visit the LinkedIn Website</cmd>
+        <substeps>
+          <substep>
+            <cmd>Open a web browser and navigate to www.linkedin.com</cmd>
+          </substep>
+        </substeps>
+      </step>
+      <step>
+        <cmd>Click on 'Join now'</cmd>
```
Comment (Eden-Miller) [inline tasks/t_linkedin_create_account.dita:17]: Can delete "on"

File: tasks/t_linkedin_create_account.dita [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      </step>
+      <step>
+        <cmd>Enter Your Personal Information</cmd>
+        <substeps>
+          <substep>
+            <cmd>Enter your First Name.</cmd>
```
Comment (Eden-Miller) [inline tasks/t_linkedin_create_account.dita:23]: "Enter" feels redundant

File: tasks/t_linkedin_create_account.dita [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <step>
+        <cmd>Confirm Your Email</cmd>
+        <substeps>
+          <substep>
+            <cmd>Check your email for a confirmation message from LinkedIn. Click the confirmation link provided in the email.</cmd>
+          </substep>
+        </substeps>
+      </step>
+      <step>
+        <cmd>Complete Your Profile by adding your information:</cmd>
```
Comment (Eden-Miller) [inline tasks/t_linkedin_create_account.dita:48]: This information (below) may be better as a different task topic.

**Review by Eden-Miller — CHANGES_REQUESTED** [review]
Overall, I think the task topic is great! I left some copy and restructuring content on the code. Please reach out if you have questions.

**Pull request merged**